src/Identity/Core/src/IdentityApiEndpointRouteBuilderExtensions.cs

```diff
@@ -90,12 +90,13 @@ public static IEndpointConventionBuilder MapIdentityApi<TUser>(this IEndpointRou
         });
 
         routeGroup.MapPost("/login", async Task<Results<Ok<AccessTokenResponse>, EmptyHttpResult, ProblemHttpResult>>
-            ([FromBody] LoginRequest login, [FromQuery] bool? cookieMode, [FromQuery] bool? persistCookies, [FromServices] IServiceProvider sp) =>
+            ([FromBody] LoginRequest login, [FromQuery] bool? useCookies, [FromQuery] bool? useSessionCookies, [FromServices] IServiceProvider sp) =>
```

Comment:
```suggestion
            ([FromBody] LoginRequest login, [FromQuery] bool? useCookies, [FromQuery] bool? usePersistentCookies, [FromServices] IServiceProvider sp) =>
```
Was session deliberate?

Reply:
> Was session deliberate?

It was. I'm going to send an email to update the API proposal. The reason is the default behavior before with just `useCookies=true` was persistent and `persistCookies=false` opted out of that so it effectively defaulted to true.

It was a horrible API, but making the simpler `useCookies=true` persist cookies by default still makes sense in my opinion.